DecipherNow/gm-fabric-dashboard
# 1109 outline on keyboard not click

Resolves #1109 

## Proposed Changes

  -Changing several onClick events and some css properties to make sure table rows are only outlined on keyboard actions and not on mouse clicks.
  -Changes took place in several places:
    JVM: routes, threads, explorer
    GO: routes, functions, explorer
    services instance view
  
## How to Test

  - Navigate to places listed above and use mouse and tab actions to compare results.
  -Goal: tabs highlight selections... clicks do not... while having same behavior in all supported browsers and passing out tests.

Tested in IE11, chrome, FF52 and latest Safari (note: In Safari, to make this work in instance table view... need to go to Safari preferences> advanced and check "press tab to highlight...")
